src/scrape_up/IPL/ipl.py

```diff
@@ -0,0 +1,37 @@
+import requests
+
+# Replace 'YOUR_API_KEY' with your actual NewsAPI.org API key
+API_KEY = 'YOUR_API_KEY'
+NEWS_API_ENDPOINT = 'https://newsapi.org/v2/top-headlines'
+IPL_QUERY = 'IPL'
+NEWS_API_PARAMS = {
+    'q': IPL_QUERY,
+    'language': 'en',
+    'apiKey': API_KEY
```

Comment:
We are building a Web scrapping-based Python package as an API alternative and here you are using API keys from the users. 

Programs demanding any kind of key will never be accepted in the projects.